tests/writer_util.py

```diff
@@ -211,16 +211,16 @@ def _convert_clob_pairs(clob_pairs):
                       precision=TimestampPrecision.SECOND, fractional_precision=1),
             b'2016-02-02T00:00:30.1-07:00'
         ),
-        # (
-        #     timestamp(2016, 2, 2, 0, 0, 30, precision=TimestampPrecision.SECOND,
-        #               fractional_seconds=Decimal('0.00001000')),
-        #     (b'2016-02-02T00:00:30.000010000-00:00', b'2016-02-02T00:00:30.000010-00:00')
-        # ),
-        # (
-        #     timestamp(2016, 2, 2, 0, 0, 30, precision=TimestampPrecision.SECOND,
-        #               fractional_seconds=Decimal('0.7e-500')),
-        #     b'2016-02-02T00:00:30.' + b'0' * 500 + b'7-00:00'
-        # )
+        (
+            timestamp(2016, 2, 2, 0, 0, 30, precision=TimestampPrecision.SECOND,
+                      fractional_seconds=Decimal('0.00001000')),
+            (b'2016-02-02T00:00:30.00001000-00:00', b'2016-02-02T00:00:30.000010-00:00')
+        ),
+        (
+            timestamp(2016, 2, 2, 0, 0, 30, precision=TimestampPrecision.SECOND,
+                      fractional_seconds=Decimal('0.7e-500')),
+            (b'2016-02-02T00:00:30.' + b'0' * 500 + b'7-00:00', b'2016-02-02T00:00:30.000000-00:00')
+        )
```

Comment:
As discussed offline, C extension only supports up to 6 timestamp precision. Mentioned this in doc and will fix this in the future release.

Same for other two uncomment tests.